flink-tests/src/test/java/org/apache/flink/test/recovery/JobManagerHAProcessFailureBatchRecoveryITCase.java

```diff
@@ -95,33 +94,21 @@
 
 	private static final FiniteDuration TestTimeOut = new FiniteDuration(5, TimeUnit.MINUTES);
 
-	private static final File FileStateBackendBasePath;
-
-	static {
-		try {
-			FileStateBackendBasePath = CommonTestUtils.createTempDirectory();
-		}
-		catch (IOException e) {
-			throw new RuntimeException("Error in test setup. Could not create directory.", e);
-		}
-	}
+	@Rule
+	public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+	
+	private File zookeeperStoragePath;
```

Comment:
instead of making this a member of `JobManagerHAProcessFailureBatchRecoveryITCase`, you should make this a parameter to `testJobManagerFailure()` since this is the only place using the variable

Reply:
while at it, you could also reduce the visibility of `testJobManagerFailure()`